code/modules/mob/dead/observer/observer.dm

```diff
@@ -245,24 +245,10 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
  * I'll make this proc global and move it to its own file in a future update. |- Ricotez - UPDATE: They never did :(
  */
 /mob/proc/brighten_color(input_color)
```

Comment:
You know this is ancient stuff when the person promising to detach the proc from mobs never delivered, and the other feller calling on that hasn't been around for about 8 years. Maybe it's time for us to actually do what our ancestors failed to.